docs/concept-workflows.md

```diff
@@ -1,30 +1,212 @@
 ---
 id: concept-workflows
-title: Workflows
+title: The concept of Workflows
 sidebar_label: Workflows
 description: Temporal core abstraction is a fault-oblivious stateful Workflow. The state of the Workflow code, including local variables and threads it creates, is immune to process and Temporal service failures.
 ---
 
 import { ResponsivePlayer } from '../src/components'
 
-## Overview
+## Focus on business logic
 
-Temporal core abstraction is a **fault-oblivious stateful Workflow**. The state of the Workflow code, including local variables and threads it creates, is immune to process and Temporal service failures.
-This is a very powerful concept as it encapsulates state, processing threads, durable timers and event handlers.
+Depending on the language, a Workflow is really just a single function or object method that orchestrates a series of actions.
+The idea is that Workflow code should be able to focus on "business logic".
+When we talk about "business logic", we are talking about the literal steps an application is taking to meet the goals of a business.
 
-## Example
+```
+Workflow {
+  ChargeCustomer()
+  SendInvoice()
+  NotifyShipping()
+  UpdateInventory()
+  ...
+}
+```
```

Comment:
what language is this meant to be in? maybe worth using just numbered bullets

Reply:
Hmm - meant to be pseudo code. But ya, I think you are right a ol is better.